app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -29,9 +29,8 @@ def perform(form526_submission_id)
       rescue => e
         # only retry if the error was raised within the "with_tracking" block
         retryable_error_handler(e) if @status_job_title
-        message = "Sidekiq job id: #{jid}. The error was: #{e.message}.<br/>" \
-                  "The backtrace was:\n #{e.backtrace.join(",<br/>\n ")}"
-        form526_submission.send_rrd_alert_email('Rapid Ready for Decision (RRD) Job Errored', message)
+        message = "Sidekiq job id: #{jid}. The error was: #{e.message}.<br/>"
+        form526_submission.send_rrd_alert_email('Rapid Ready for Decision (RRD) Job Errored', message, e)
```

Comment:
I'm curious if the `retryable_error_handler` is referring to a different one than the one in submit_form_526. If it's the same, would it also need the submission argument? Would calling that and send_rrd_alert_email here send two alert emails?

Reply:
The `retryable_error_handler(e)` on line 31 calls `JobTracker#retryable_error_handler`, not `SubmitForm526#retryable_error_handler`. The `Form526BaseJob` and `SubmitForm526` are different jobs.